pkg/codegen/internal/test/testdata/dashed-import-schema/nodejs/tree/v1/rubberTree.ts

```diff
@@ -61,16 +61,16 @@ export class RubberTree extends pulumi.CustomResource {
             if ((!args || args.type === undefined) && !opts.urn) {
                 throw new Error("Missing required property 'type'");
             }
-            inputs["container"] = args ? args.container : undefined;
-            inputs["diameter"] = (args ? args.diameter : undefined) ?? 6;
-            inputs["farm"] = (args ? args.farm : undefined) ?? "(unknown)";
-            inputs["size"] = (args ? args.size : undefined) ?? "medium";
-            inputs["type"] = (args ? args.type : undefined) ?? "Burgundy";
+            resourceInputs["container"] = args ? (args.container ? pulumi.output(args.container).apply(inputs.containerArgsProvideDefaults) : undefined) : undefined;
```

Comment:
Can't seem to figure out where `inputs` is coming from in `inputs.containerArgsProvideDefaults` after `let inputs: pulumi.Inputs = {};` is removed.

Reply:
Ok never mind, `import { input as inputs, output as outputs, enums } from "../../types";`. So previously there was shadowing. Unfortunate.